_gitbook/conventions/coding_style.md

```diff
@@ -0,0 +1,128 @@
+# Coding Style
+
+This style is used in the standard library. You can use it in your own project to make it familiar to other developers.
+
+## Naming
+
+__Class names__ are camelcased. For example:
```

Comment:
Yes, `type` would be better than `class` since Crystal features classes, structs, modules and enums... (and libs)